### PR TITLE
Feature/output additional datasets

### PR DESCRIPTION
**Pull request by seb231**
Allow both the CCM workflow and demo tool to output multiple datasets (to distinct csv's for the latter).

### CODE REVIEW

File: src/witan/models/dem/ccm/core/projection_loop.clj [hunk 1]
```diff
@@ -144,5 +144,7 @@
   "Returns the population field"
   {:witan/name :ccm-core-out/population
    :witan/version "1.0.0"
-   :witan/input-schema {:population PopulationSchema}}
+   :witan/input-schema {:population PopulationSchema :net-migration NetMigrationSchema
+                        :births-by-age-sex-mother BirthsAgeSexMotherSchema
+                        :births BirthsBySexSchema :deaths DeathsOutputSchema}}
```
Comment (sunnytownsend) [inline src/witan/models/dem/ccm/core/projection_loop.clj:148]: these look good to me @seb231 - good call on included both forms of birth data. We could add more too e.g. full migration projections or projected rates, but I think this is great for starters.

File: src-cli/witan/models/run_models.clj [hunk 2]
```diff
@@ -243,6 +243,11 @@
                                                                        (.getMessage e))))))
         (map? input) input))
 
+(defn output-to-csv [data gss-code dataset key output-projections]
```
Comment (elisehuard) [inline src-cli/witan/models/run_models.clj:254]: lgtm

**Review by elisehuard — APPROVED** [review]

File: src/witan/models/dem/ccm/core/projection_loop.clj (hunk 1 above)
Comment (sunnytownsend) [reply src/witan/models/dem/ccm/core/projection_loop.clj:148]: after discussion it was agreed that the outputs should be `:births, :deaths, :net-migration and :population`

**Comment by sunnytownsend** [comment]
:shipit:

**Pull request merged**